topology/probes/nsm/nsm.go

```diff
@@ -70,30 +71,45 @@ func NewNsmProbe(g *graph.Graph) (*Probe, error) {
 	return probe, nil
 }
 
+func getK8SConfig() (*rest.Config, error) {
```

Comment:
this duplicates code already found in topology/probes/k8s ... better you generalize and reuse the code found there